src/library/simplified/extension/doubleclick-src-cat-type-PHIS.js

```diff
@@ -0,0 +1,306 @@
+if(b.siteId && utag.isPIS_FH() ){
```

Comment:
@trungguyen yes it includes FHC as well.
![image](https://user-images.githubusercontent.com/21168840/28656455-bde5c54e-72bf-11e7-8213-754a8fd40f3b.png)